carta/session.py

```diff
@@ -391,6 +391,31 @@ def append_image(self, path, hdu="", complex=False, expression=ArithmeticExpress
             Arithmetic expression to use if appending a complex-valued image. The default is :obj:`carta.constants.ArithmeticExpression.AMPLITUDE`.
         """
         return Image.new(self, path, hdu, True, complex, expression)
+    
+    def load_stokes_hypercube(self, output_direcory, directories, files, polarization_type, hdu="", append=False):
+        """Open or append a new Stokes hypercube with the selected Stokes images.
+
+        Parameters
+        ----------
+        output_direcory : {0}
```

Comment:
why do we need this `output_directory` parameter? A Stokes hypercube is a virtual image which is constructed from 2-4 on-disk images.

Reply:
Since the `openConcatFile` and `appendConcatFile` need that parameter (the `directory` of the image shown below):
![image](https://user-images.githubusercontent.com/106953609/233037243-da0ff060-6699-4caf-a4c5-64b1464afa7b.png)
Actually I don't know why this parameter is needed here, since the directories of the input images are given with the `IStokesFile` (see [this line](https://github.com/CARTAvis/carta-frontend/blob/dev/src/stores/AppStore/AppStore.ts#L598)). And if I set this `output_directory` to other directory (not the same directory with the stokes image), the hypercube is still loaded out. Then when I try to `Save Image` in the `File` menu, the default directory is the one I just set. Thus I recognize it as the output directory.